### PR TITLE
`core` broken out

### PR DESCRIPTION
**Pull request by wjohnsto**
This PR satisfies #198.

NOTE: There is no name for the framework yet, so we are going to have to use a temporary alias while we wait on naming.

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
+{
+  "name": "@wpengine/headless",
```
Comment (blakewilson) [inline packages/core/package.json:2]: Lerna is failing due to two packages (`packages/core/package.json`, `packages/headless/package.json`) with the same name.

Maybe for now we could go with something like this until we finalize a name?

```suggestion
  "name": "@wpengine/headless-core",
```

**Review by blakewilson — COMMENTED** [review]
Only thing I see is the core `package.json` name. That will need to be updated in order for Lerna to run. I've got this set to `@wpengine/headless-core` in the react package for now until we come up with a final name.

**Comment by wjohnsto** [comment]
> 
> 
> Only thing I see is the core `package.json` name. That will need to be updated in order for Lerna to run. I've got this set to `@wpengine/headless-core` in the react package for now until we come up with a final name.

I saw you renamed it so I figured I would let you include that as part of your PR for the react components.

**Comment by blakewilson** [comment]
@wjohnsto Perfect! LGTM then.

**Review by blakewilson — APPROVED** [review]

**Pull request merged**